02_knight_and_princess/tevfik_aksoy/main.cpp

```diff
@@ -0,0 +1,145 @@
+#include <fstream>
+#include <iostream>
+#include <set>
+#include <string>
+#include <tuple>
+#include <vector>
+
+using std::cout;
+using std::endl;
+using std::get;
+using std::to_string;
+using std::vector;
+
+using NodeId = int;
+using EdgeId = int;
+
+// Tuple of <int,int> for realization of vector of vectors of <int,int>
+using dest_edge_id = std::tuple<int, int>;

```

Comment:
Наверное, это должны быть не `int`, a `NodeId`?
```suggestion
using dest_edge_id = std::tuple<NodeId, NodeId>;
```
Все еще из названия не понимаю, что означает этот тип. "Destination Edge Id"? Зачем он тогда хранит 2 значения, если `id` должен быть одним значением?

Можешь объяснить свою логику?

Reply:
То есть этот тип хранит DestinationID, EdgeID